jenkins_jobs/modules/triggers.py

```diff
@@ -240,6 +251,26 @@ def build_gerrit_skip_votes(xml_parent, data):
         XML.SubElement(skip_vote_node, tag_name).text = str(setting).lower()
 
 
+def build_cancellation_policy(xml_parent, data, plugin_ver):
+    if plugin_ver >= pkg_resources.parse_version("2.32.0"):
+        options = [
+            ("enabled", "enabled"),
```

Comment:
I'm not sure about the default value here (set to False below). See https://github.com/jenkinsci/gerrit-trigger-plugin/blob/f182f4240beeb4dad26f4e98e6fad07bb0212e32/src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/data/BuildCancellationPolicy.java#L56

Can you check in live Jenkins what the defaults are?

Reply:
I see it as disabled on the jenkins instance. Maybe the `DataBoundConstructor ` annotation shows the defaults when it's disabled? Because the default constructor is empty.

Reply:
I checked how it works in practice. When you check "Build Current Patches Only" then the following section is added to XML:
```
<buildCancellationPolicy>
    <enabled>true</enabled>
    ...
</buildCancellationPolicy>
  ```
When you don't check "Build Current Patches Only" then the whole piece of XML is missing in config XML. So I think:
1) `<enabled>` should not be configurable
2) `<buildCancellationPolicy>` should be added to XML conditionally based on `build-cancellation-policy` YAML property